File: LR_2/2.13.pas
{ Задание 2.13 }

var
  elIndex: integer;
  value, radius, area: real;

begin
  writeln('Введите номер элемента:');
  writeln('1 - Радиус');
  writeln('2 - Диаметр');
  writeln('3 - Длина окружности');
  readln(elIndex);
  
  if elIndex = 1 then
  begin
    writeln('Введите радиус:');
      readln(value);
      radius := value;
  end;
  
  if elIndex = 2 then
  begin
      writeln('Введите диаметр:');
      readln(value);
      radius := value / 2;
  end;

  if elIndex = 3 then
  begin
      writeln('Введите длину окружности:');
      readln(value);
      radius := value / (2 * pi);
  end;  
  
  area := pi * radius * radius;
  area := (round(area*1000))/1000;
  writeln('Площадь круга: ', area);
end.

File: LR/LR_2/2.13.py
# Задание 2.13

import math

el_index = int(input("Введите номер элемента:\n1 - Радиус\n2 - Диаметр\n3 - Длина окружности\n"))

if el_index == 1:
    value = int(input("Введите радиус:\n"))
    radius = value
elif el_index == 2:
    value = int(input("Введите диаметр:\n"))
    radius = value / 2
elif el_index == 3:
    value = int(input("Введите длину окружности:\n"))
    radius = value / (2 * math.pi)

area = math.pi * radius ** 2
area = round(area, 3)

print("Площадь круга:" + str(area))